### ISSUE TITLE
Radio button ignores disabled state

### ISSUE THREAD

**Issue by ahumellihuk** [gwtbootstrap3/gwtbootstrap3#407]
Value change events are triggered even when the radio button is disabled.

**Comment by slugmandrew** [comment]
I can work out how to stop the `ValueChangeEvent` firing but I can't stop the active style being set for some reason :confused:

**Comment by sjardine** [comment]
OK.  Please describe exactly the problem.  Could you provide an example for me to work with?  The easiest thing to do is fork the seed project and create the demo there. 

Thanks!

**Comment by sjardine** [comment]
Can you please provide additional detailed information?

**Comment by slugmandrew** [comment]
It wasn't me that opened the issue, but I did have a go at fixing it. As I said I managed to fix it so that when pressed the radio button did not fire the `ValueChangeEvent`, but it still *looked like* it had fired the event, which would be more confusing for the user.

I'll try to fork the seed project sometime in the next couple of weeks and set up an example. Unfortunately I'm very busy atm.

**Comment by sjardine** [comment]
I can create the example if you could give help explain the setup.  I haven't been able to duplicate.

**Comment by vlahoda** [comment]
This error is reproducible even in the demo at http://gwtbootstrap3.github.io/gwtbootstrap3-demo/#buttonGroups . In the Radio buttons example at the very bottom of that page it is possible to click on the disabled Option4, which in turn gets assigned the style "active".

**Comment by nemoo2580** [comment]
It's also a bootstrap bug:
https://github.com/twbs/bootstrap/issues/16703

**Comment by crevete** [comment]
You are right, this is a bootstrap bug, but still not fixed in 3.3.7. Hope it will be fixed soon...
